Source/Operations/AddTableOperation.cs

```diff
@@ -115,7 +115,7 @@ protected override MergeResult DoMerge(Operation operation)
 					if( column != null )
 					{
 						column.DefaultExpression = addDefaultConstraintOp.Expression;
-						operation.Disabled = true;
+                        addDefaultConstraintOp.Disabled = true;

```

Comment:
Indentation looks off again. Is this just GitHub showing it this way?

Reply:
No. Visual Studio doesn't do a good job of keeping consistent indent characters.